test/book/specialized/brandhorst-zach-fibration-hopping/vinberg_2.jlcon

```diff
@@ -122,17 +122,17 @@ julia> g, phi1 = two_neighbor_step(Y1, fibers[2][I]); g
 
 julia> kt2 = base_ring(parent(g)); P = kt2.([0,0]);
 
-julia> Y2, phi2 = elliptic_surface(g, P); Y2
+julia> Y2, phi2 = elliptic_surface(g, P; minimize=false); Y2
```

Comment:
not sure in which Oscar versions this works. It is a recent addition. in 5dcfffb9553

Reply:
I think it should work in 1.1.1, but not in 1.1.0.